build.gradle

```diff
@@ -86,110 +87,73 @@ test {
 }
 
 publishing {
-  afterEvaluate {
-    def pluginUrl = 'https://github.com/sonatype-nexus-community/scan-gradle-plugin'
-
-    pluginBundle {
-      website = pluginUrl
-      vcsUrl = pluginUrl
-      description = pluginDescription
-      tags = ['sonatype', 'scan', 'dependencies', 'ossindex', 'iq server']
-      System.setProperty('gradle.publish.key', System.getenv('GRADLE_PUBLISH_KEY') ?: '')
-      System.setProperty('gradle.publish.secret', System.getenv('GRADLE_PUBLISH_SECRET') ?: '')
-    }
-
-    release {
-      // avoid infinite CircleCI builds triggered by release commits
-      preCommitText = '[skip ci] '
-      buildTasks = ['assemble']
-    }
-
-    def sonatypeUsername = System.getenv('SONATYPE_USERNAME')
-    def sonatypePassword = System.getenv('SONATYPE_PASSWORD')
-
-    def serverUrlBase = 'https://repository.sonatype.org:443/service/local/'
-
-    repositories {
-      maven {
-        name = 'rso'
-        url = serverUrlBase + 'staging/deploy/maven2/'
-        credentials {
-          username = sonatypeUsername
-          password = sonatypePassword
+  publications {
+    mavenJava(MavenPublication) {
+      from components.java
+      groupId = 'org.sonatype.gradle.plugins'
+      artifactId = project.name
+      version = project.version.toString()
+
+      pom {
+        name = project.name
+        description = pluginDescription
+        url = 'https://github.com/sonatype-nexus-community/scan-gradle-plugin'
+        organization {
+          name = 'Sonatype'
+          url = 'https://www.sonatype.org'
```

Comment:
Is this correct? `.org` doesn't go anywhere for me, however `https://www.sonatype.com` does.